dali/operators/decoder/nvjpeg/nvjpeg_decoder_decoupled_api.h

```diff
@@ -106,7 +113,8 @@ class nvJPEGDecoder : public Operator<MixedBackend>, CachedDecoderImpl {
     }
 
     if (try_init_hw_decoder &&
-        nvjpegCreate(NVJPEG_BACKEND_HARDWARE, NULL, &handle_) == NVJPEG_STATUS_SUCCESS) {
+        nvjpegCreateEx(NVJPEG_BACKEND_HARDWARE, NULL, NULL, nvjpeg_flags, &handle_)
+                                                                        == NVJPEG_STATUS_SUCCESS) {
```

Comment:
weird indentation

Reply:
does the HW decoder support this? Something to check

Reply:
> weird indentation

Done